packages/shared/src/components/tooltips/LazyTooltip.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState, ReactElement, Ref } from 'react';
+import Tippy, { TippyProps } from '@tippyjs/react';
+import classNames from 'classnames';
+import styles from './LazyTooltip.module.css';
+import { isTouchDevice } from '../../lib/tooltip';
+import { isTesting } from '../../lib/constants';
+import {
+  TooltipContainer,
+  TooltipContainerProps,
+  TooltipPosition,
+} from './TooltipContainer';
+
+const DEFAULT_DELAY_MS = 400;
+const DEFAULT_OUT_ANIMATION = 200;
+
+export const getShouldLoadTooltip = (disableTooltip?: boolean): boolean =>
+  !isTouchDevice() && !isTesting && !disableTooltip;
+
+export interface LazyTooltipProps extends TippyProps {
+  container?: Omit<TooltipContainerProps, 'children'>;
+  placement?: TooltipPosition;
+}
+
+export default function LazyTippy(
```

Comment:
Why is it called `LazyTippy`? Where's the lazy part?
Also I'm not sure exactly what's the exact purpose of this component

Reply:
I think I should rename this better to make its purpose much clearer.